charts/karmada/values.yaml

```diff
@@ -500,7 +496,7 @@ kubeControllerManager:
     rollingUpdate:
       maxUnavailable: 0
       maxSurge: 50%
-
+  controllers: namespace,garbagecollector,serviceaccount-token,ttl-after-finished,bootstrapsigner,csrapproving,csrcleaner,csrsigning
```

Comment:
Add a blank line here.

Reply:
Got it.

Reply:
Sorry, I mean add a blank between it and the etcd config.

Reply:
:smile::smile:
Rebased it.